### PR TITLE
Ads: Update Settings Card to new UI

### PR DESCRIPTION
**Pull request by dbspringer**
Updating settings per: https://github.com/Automattic/jetpack/issues/6224

screen shot 2017-02-07 at 3 10 54 pm

### CODE REVIEW

File: _inc/client/traffic/ads.jsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+	React.createClass( {
+
+		/**
+		 * Get options for initial state.
+		 *
+		 * @returns {{enable_header_ad: Boolean}}
+		 */
+		getInitialState() {
+			return {
+				enable_header_ad: this.props.getOptionValue( 'enable_header_ad', 'wordads' ),
```
Comment (eliorivero) [inline _inc/client/traffic/ads.jsx]: Unneeded comma at the end of this line.

**Comment by dbspringer** [comment]
@eliorivero does it make sense to merge this into `feature/settings-overhaul` now, or wait until more further development has occured?

**Comment by eliorivero** [comment]
Yes, it's ok to merge it. We're iterating this rapidly and trying to merge as fast as possible since we have a lot of moving pieces like the paid banners, which actually have to be integrated in this card and it has to be disabled based in the plan.

My only concern is that there are two things in the implementation below that are not like the mockup in https://github.com/Automattic/jetpack/issues/6224 and should be addressed before merging.

captura de pantalla 2017-02-08 a las 19 28 24

- the text _By activating ads..._ is missing the class `jp-form-setting-explanation` that will make it italic and in a different color
    captura de pantalla 2017-02-08 a las 19 34 46

- the link View your earnings is not indented.
    It's enough if you move it inside the `fieldset`, it will be indented.
    captura de pantalla 2017-02-08 a las 19 14 02

**Comment by dbspringer** [comment]
Thanks @eliorivero, I made those updates.

**Comment by dbspringer** [comment]
Though just to be clear, @MichaelArestad did you intend for the link to be indented like that? Seems out of place with the other cards' footer links.

screen shot 2017-02-08 at 2 45 36 pm

**Comment by MichaelArestad** [comment]
@dbspringer I did mean for it to be indented. It matches the `View your Email Followers` link in Subscriptions.

**Comment by eliorivero** [comment]
Looks good! merging so @zinigor can handle their integration with banners since this is a paid feature.

**Pull request merged**